.github/workflows/release.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: npm ci
       - name: Release
         run: |
-          msg="${{ github.event.head_commit.message }}"
+          msg='${{ github.event.head_commit.message }}'
```

Comment:
Let's hope that github will still evaluate it's variables :D furthermore, in general, single ticks could be used in the message to trick them again. but as long as only dedicated people can run this... we should be safe :eyes:

Reply:
According to the docs, GH variables are evaluated in the workflow file before it is actually sent to a runner, so I doubt it cares about what is around the variables. 

Also yes the single tick thing iiiis a good point hmmmm